cpp/include/cudf/io_types.h

```diff
@@ -23,27 +23,47 @@
  */
 #pragma once
 
+typedef enum 
+{
+  FILE_PATH,
+  BUFFER,
+  URL
+} resource_type;
+
+typedef struct 
+{  
+  char			*data;			/**< pointer to a host memory buffer	*/
+  size_t		size;			/**< size of the buffer, in bytes		*/
+} buffer_desc;
+
+typedef struct
```

Comment:
block comment for this struct.

Reply:
removed this struct as suggested by @j-ieong